### PR TITLE
TCM-109 | Build dummy trainer screen

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Completed dummy trainer screen.
- [x] Added trainer to test utility.

### Issue

[TCM-109](https://hyfnla.atlassian.net/browse/TCM-109)

### Video/Screenshot

BEFORE SCROLL

trainer_1

AFTER SCROLL

trainer_2

DEMO

https://user-images.githubusercontent.com/30659854/119189091-43bf6c00-ba84-11eb-9afb-45ad9336ccd2.mp4

### CODE REVIEW

**Comment by halilyesilyurt** [comment]
added demo.

File: app/src/main/res/layout/fragment_trainer.xml [hunk 1]
```diff
@@ -0,0 +1,262 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.trx.consumer.common.CommonView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:backgroundColor="@color/white">
+
+    <ScrollView
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_trainer.xml:9]: Let's make this a `NestedScrollView`

File: app/src/main/res/layout/fragment_trainer.xml [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+                    android:text="Arnold B."
+                    android:textColor="@color/blackRussian"
+                    android:textSize="32sp"
+                    app:layout_constraintTop_toBottomOf="@id/imgPhoto"
+                    tools:text="Arnold B." />
+
+
+            </com.trx.consumer.common.CommonView>
+
+            <com.trx.consumer.common.CommonLabel
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_trainer.xml:74]: Followup task: Need banners and trainer tagline above

File: app/src/main/res/layout/fragment_trainer.xml [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+
+            <View
+                android:id="@+id/vLineOne"
+                android:layout_width="match_parent"
+                android:layout_height="1dp"
+                android:layout_marginTop="20dp"
+                android:background="@color/greyLight"
+                app:layout_constraintTop_toBottomOf="@+id/lblSummary" />
+
+            <com.trx.consumer.common.CommonLabel
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_trainer.xml:105]: Followup task: We have a re-usable `layout_live_workout_section` we can `<include` here

File: app/src/main/res/layout/fragment_trainer.xml [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+            <View
+                android:id="@+id/vLineTwo"
+                android:layout_width="match_parent"
+                android:layout_height="1dp"
+                android:layout_marginTop="15dp"
+                android:background="@color/greyLight"
+                app:layout_constraintTop_toBottomOf="@+id/rvUpcomingClasses" />
+
+            <com.trx.consumer.common.CommonLabel
+                android:id="@+id/lblServices"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_trainer.xml:152]: Followup task: Let's center this between line above and cells below

File: app/src/main/res/layout/fragment_trainer.xml [hunk 5]
```diff
@@ -0,0 +1,262 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.trx.consumer.common.CommonView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_trainer.xml:2]: For followup task: Use re-usable layout for the sections wherever possible

File: app/src/main/res/layout/fragment_trainer.xml [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+                android:padding="15dp"
+                android:text="@string/trainer_button_see_all"
+                android:textAllCaps="true"
+                android:textColor="@color/blackRussian"
+                android:textSize="12sp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/vLineThree" />
+
+            <com.trx.consumer.common.CommonRecyclerView
+                android:id="@+id/rvOnDemandClasses"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_trainer.xml:215]: For followup task: Remove `marginStart` and give the row view the margin start instead. Same for the `services` RecyclerViews

**Review by VivekV95 — APPROVED** [review]

**Review by onodude — APPROVED** [review]

**Pull request merged**